stable/jenkins/values.yaml

```diff
@@ -27,6 +27,12 @@ Master:
   SlaveListenerPort: 50000
   LoadBalancerSourceRanges:
   - 0.0.0.0/0
+# List of plugins to be install during Jenkins master start
+  InstallPlugins:
+      - kubernetes:0.11 
+      - workflow-aggregator:2.5
+      - credentials-binding:1.11
+      - git:3.2.0
```

Comment:
Ok, no problem.  It's a small plugin and not obvious (i.e. from Jenkins build log error messages) it is necessary to install when pulling Jenkinsfiles utilizing `@Library('github.com/YourRepo/YourProject@master')` in a GitHub repo, so just thought it could be helpful for people by default.